docs/en/latest/how-to-build.md

```diff
@@ -21,36 +21,37 @@ title: How to build Apache APISIX
 #
 -->
 
-## Step 1: Install dependencies
-
-The Apache APISIX runtime environment requires dependencies on NGINX and etcd.
-
-Before installing Apache APISIX, please install dependencies according to the operating system you are using. We provide the dependencies installation instructions for **CentOS7**, **Fedora 31 & 32**, **Ubuntu 16.04 & 18.04**, **Debian 9 & 10**, and **MacOS**, please refer to [Install Dependencies](install-dependencies.md) for more details.
-
-## Step 2: Install Apache APISIX
+## Step 1: Install Apache APISIX
 
 You can install Apache APISIX via RPM Repository, RPM package, Docker, Helm Chart, and source release package. Please choose one from the following options.
 
 ### Installation via RPM Repository(CentOS 7)
 
-This installation method is suitable for CentOS 7. For now, the Apache APISIX RPM repository for CentOS 7 is already supported. Please run the following commands to install the repository and Apache APISIX.
+This installation method is suitable for CentOS 7.
+
+If the official OpenResty repository is not installed yet, the following command will help you automatically install both the repositories of OpenResty and Apache APISIX.
 
 ```shell
-sudo yum-config-manager --add-repo https://repos.apiseven.com/packages/centos/apache-apisix.repo
-# View the information of the latest apisix package
-sudo yum info -y apisix
+$ sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
```

Comment:
I think
`sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm` is the same as
`sudo yum-config-manager --add-repo https://repos.apiseven.com/packages/centos/apache-apisix.repo`?

@imjoey Please correct me if I am wrong.

Reply:
> I think `sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm` is the same as `sudo yum-config-manager --add-repo https://repos.apiseven.com/packages/centos/apache-apisix.repo`?
> 
> @imjoey Please correct me if I am wrong.

@spacewander not exactly. Except for adding the apache-apisix.repo,  installing the rpm will also add the official openersty.repo.

Reply:
Thanks for your explanation. 😄